clients/venice-producer/src/main/java/com/linkedin/venice/producer/online/ProducerTool.java

```diff
@@ -76,8 +74,7 @@ public class ProducerTool {
       .addOption(HELP_OPTION);
 
   public static void main(String[] args) throws Exception {
-    // Disable Logging
-    Configurator.setAllLevels(LogManager.getRootLogger().getName(), Level.OFF);
+    LogConfigurator.disableLog();
```

Comment:
Can we try to use `log4j2.properties` file to control the logging output instead of switching it through the code

Reply:
I'm not sure how to specify different properties files for the shadow jar vs the regular jar and I'm not too keen on spending much cycles for it. These things generally tend to end up being rabbit holes. 

This way, we can control the log behavior in a quick (and admittedly, dirty) way

Reply:
We could use a props file for the shadow jar. For regular jar-calling applications can control logging behavior as per their need.